tests/unittests/test_logger.py

```diff
@@ -0,0 +1,78 @@
+import unittest
+from unittest.case import TestCase
```

Comment:
We can remove the unnecessary import of `TestCase` as the test case uses `unittest.TestCase`.

Reply:
done